bftengine/tests/bcstatetransfer/test_app_state.hpp

```diff
@@ -0,0 +1,89 @@
+// Concord
+//
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#ifndef BFTENGINE_TESTS_BCSTATETRANSFER_TEST_APP_STATE_HPP_
+#define BFTENGINE_TESTS_BCSTATETRANSFER_TEST_APP_STATE_HPP_
+
+#include <cassert>
+#include <cstring>
+#include <unordered_map>
+#include "SimpleBCStateTransfer.hpp"
+
+// This should be the same as test config
+const uint32_t kMaxBlockSize = 1024;
```

Comment:
Should kMaxBlockSize be visible outside SimpleBlockchainStateTransfer?

Reply:
Yes, it is a parameter to SimpleBlockchainStateTransfer.
(see SimpleBlockchainStateTransfer::Config)